config/locales/en.txt

```diff
@@ -1937,11 +1926,7 @@
   rss_created_at: "[TYPE] Created"
   rss_changed: "[TYPE] Changed"
   rss_destroyed: "[TYPE] Destroyed"
-  rss_consensus_changed: "Consensus changed from: [old]"
   rss_consensus_reached: Consensus established
-  rss_naming_created_at: "Name proposed by [user]: [name]"
-  rss_naming_destroyed: "Name proposal destroyed by [user]: [name]"
-  rss_naming_updated: "Name proposal updated by [user]: [name]"
```

Comment:
Do you remember why these are all being removed?  Are these no longer produced anywhere?  Have you checked that they are not present in the existing database, too?

Reply:
For all the removals, these came from an email you sent me. I triple checked them before committing the deletes, but that doesn't mean i didn't mess up. Will post your list here as soon as I find it.

Reply:
@pellaea This was your proposed list:

> Change these:
>
>   # log_consensus_changed: "Consensus changed from: [old]" <--- maybe not, see below
>   log_naming_created_at: "Name proposed by [user]: [name]"
>   log_naming_destroyed: "Name proposal destroyed by [user]: [name]"
>   log_naming_updated: "Name proposal updated by [user]: [name]"
> 
> Delete these:
> 
>   rss_consensus_changed
>   rss_naming_created_at
>   rss_naming_destroyed
>   rss_naming_updated
>   log_object_added_no_user
>   log_object_created_no_user
>   log_object_destroyed_no_user
>   log_object_merged_no_user
>   log_object_moved_no_user
>   log_object_removed_no_user
>   log_object_updated_no_user
>   log_image_created0
>   log_image_destroyed0
>   log_image_removed0
>   log_image_reused0
>   log_image_updated0